### DISCUSSION TITLE
npx directus constantly ask for an upgrade

### DISCUSSION THREAD

**Discussion by cybermedi** [directus/directus#22319] [Feature Requests]
### Summary

I try to run e.g.: 
```
npx directus --version
Need to install the following packages:
directus@10.10.7
Ok to proceed? (y) n
```

but 
```
npm update directus
up to date, audited 12 packages in 525ms
found 0 vulnerabilities
```

any one has an idea?

### Motivation

just stop the directus

### Detailed Design

.

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

.

### Alternatives

.

### Adoption Strategy

.

**Comment by br41nslug** [comment]
Do you have the correct version installed in the package.json of the folder you're running that command in? If `npx` cant find the local installation it will always try to install it with this message.
